minigalaxy/download_manager.py

```diff
@@ -568,7 +623,7 @@ def __is_same_download_as_before(self, download):
         # Check first part of the file
         resume_header = {'Range': 'bytes=0-{}'.format(size_to_check - 1)}  # range header is index-0-based
         with self.session.get(download.url, headers=resume_header, stream=True, timeout=30) as download_request:
-            if not download_request.status_code == codes.ok:
+            if not 200 <= download_request.status_code < 300:
```

Comment:
This can be `if not download_request.ok:`: https://docs.python-requests.org/en/latest/api/#requests.Response.ok